examples/detective/member.yaml

```
@@ -0,0 +1,30 @@
+apiVersion: detective.aws.upbound.io/v1beta1
+kind: Member
+metadata:
+  annotations:
+    upjet.upbound.io/manual-intervention: "Requires AccountId and AccountEmail to replaced manually."
+  labels:
+    testing.upbound.io/example-name: example
+  name: example
+spec:
+  forProvider:
+    accountId: <AccountId>
+    disableEmailNotification: true
+    emailAddress: <AccountEmail>
```

```suggestion
    emailAddress: ${data.aws_account_root_user_email}
```

Similarly, to avoid manual intervention.

---

It could not be changed with emailAddress: ${data.aws_account_root_user_email}, <AccountEmail> - it is not current(root) account email, it is any another AWS user we will add as a member. For this test we user free tire user.

---

done